examples/auth-rust-demo/tests/src/tests.rs

```diff
@@ -0,0 +1,22 @@
+// use super::*;
```

Comment:
We need to demonstrate how to test ckb-auth in a capsule-generated project, rather than commenting out the generated test files.

Reply:
The capsule is mainly used for building contracts. The tests for ckb-auth, especially the e2e tests are very complicated. It can't be handled by capsule.

Reply:
I think this demo project `auth-rust-demo` is generated by capsule, right? It should demonstrate how to use and test `ckb-auth-rs` crate in a rust contract project.